lib/createDIContainer.js

```diff
@@ -142,12 +144,15 @@ async function createDIContainer(options) {
     transactionalDataProvider: asClass(TransactionalMachineDataProvider),
 
     deliverTxDpp: asFunction(transactionalDataProvider => (
```

Comment:
We need two, isolated `deliverTxDpp` for `createFromSerialzied` and a normal one for `validateData`